userspace/libsinsp/test/public_sinsp_API/interesting_syscalls.cpp

```diff
@@ -112,12 +112,12 @@ TEST(interesting_syscalls, names_sc_set_names_corner_cases)
 	/* INCONSISTENCY: `names_to_sc_set` is converting event names to ppm_sc, but this was not its original scope, the original scope was to convert sc_names -> to sc_set  */
 	std::unordered_set<std::string> event_names{"accept", "execve", "syncfs", "eventfd", "umount", "pipe", "signalfd", "umount2"};
 	auto sc_set = libsinsp::events::names_to_sc_set(event_names);
-	libsinsp::events::set<ppm_sc_code> expected_sc_set{PPM_SC_ACCEPT, PPM_SC_ACCEPT4, PPM_SC_EXECVE, PPM_SC_SYNCFS, PPM_SC_EVENTFD, PPM_SC_EVENTFD2, PPM_SC_UMOUNT, PPM_SC_PIPE, PPM_SC_PIPE2, PPM_SC_SIGNALFD, PPM_SC_SIGNALFD4, PPM_SC_UMOUNT2};
+	libsinsp::events::set<ppm_sc_code> expected_sc_set{PPM_SC_ACCEPT, PPM_SC_ACCEPT4, PPM_SC_EXECVE, PPM_SC_SYNCFS, PPM_SC_EVENTFD, PPM_SC_UMOUNT, PPM_SC_PIPE, PPM_SC_SIGNALFD, PPM_SC_UMOUNT2};
```

Comment:
now we don't have anymore these kinds of inconsistencies, we don't expect anymore these ppm_sc for example when we provide `pipe` event we should enable only `PPM_SC_PIPE` not also `PPM_SC_PIPE2`